lib/cequel/record/tasks.rb

```diff
@@ -63,12 +63,15 @@ def drop!
     puts "Dropped keyspace #{Cequel::Record.connection.name}"
   end
 
+  def models_dir_path
+    ENV['CEQUEL_MODELS_PATH'] || "#{File.expand_path('app/models', project_root)}/"
```

Comment:
I'm saying that i like `Hash#fetch` and `Pathname#+` better than `Hash#[]` and `File.expand_path`. 

It is a small thing, on which reasonable people can disagree, and is not necessary to merge this PR.